### ISSUE TITLE
Add XFCE compatible desktop icon

### ISSUE THREAD

**Issue by jsno8192** [brickviking/night-hawk#22]
Requested by giantclambake (aka wotnot). He has strongly recommended I include a XFCE compatible decktop icon in 4.0. I will look into it tomorrow (no promises !):

Create desktop_icon.png (GCB's recommendation is a 606 droid)

Get the cmake script to create and copy this file to ~/Desktop/Nighthawk.desktop or .../share/applications/Nighthawk.desktop:

--$--------------------------
> [Desktop Entry]
> Version=1.0
> Type=Application
> Name=Nighthawk
> Comment=
> Exec=.../bin/nighthawk
> Icon=.../share/nighthawk/Nighthawk/logo.png
> Path=
> Terminal=false
> StartupNotify=false
--$--------------------------

**Comment by brickviking** [comment]
About the only thing I might comment on in regard to this is: where would the debug log go? Currently debug goes to stdout. A program launched from the desktop doesn't have its stdout/stderr redirected to files.

**Comment by jsno8192** [comment]
You are right. It's not kosher. It should output to an optional file. We can change that in 4.1dev if you like ?
